src/main/java/com/example/rets_api/controller/RetsController.java

```diff
@@ -61,6 +61,20 @@ public ResponseEntity<PropertyDTO> getPropertiesById(@PathVariable("id") Long pr
         return handleResponse(propertyDTO);
     }
 
+    @ApiOperation(value = "delete a property by id")
```

Comment:
Please use capital letter (Delete a property by id), just to keep the pattern.

Reply:
done